File: src/test/java/ru/tinkoff/load/jdbc/test/scenarios/JdbcBasicSimulation.java
package ru.tinkoff.load.jdbc.test.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;
import static io.gatling.javaapi.core.CoreDsl.scenario;
import static ru.tinkoff.load.jdbc.test.cases.JdbcActions.*;

public class JdbcBasicSimulation {
    public static ScenarioBuilder scn = scenario("JDBC scenario")
            .exec(createTable())
            .exec(createprocedure())
            .exec(insertTest())
            .exec(callTest())
            .exec(batchTest())
            .exec(selectTT())
            .exec(selectTest())
            .exec(selectAfterBatch())
            .exec(checkTestTableAfterBatch())
            ;
}

File: src/test/kotlin/ru.tinkoff.load.jdbc.test/scenarios/JdbcBasicSimulation.kt
package ru.tinkoff.load.jdbc.test.scenarios

import ru.tinkoff.load.jdbc.test.cases.JdbcActions

object JdbcBasicSimulation {
    var scn: io.gatling.javaapi.core.ScenarioBuilder = io.gatling.javaapi.core.CoreDsl.scenario("JDBC scenario")
            .exec(JdbcActions.createTable())
}